sdk/core/core/inc/az_platform.h

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef _az_PLATFORM_H
+#define _az_PLATFORM_H
+
+#include <az_result.h>
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <_az_cfg_prefix.h>
+
+/**
+ * @brief az_platform_clock_msec gets platform clock in milliseconds.
+ * The moment of time where clock starts is undefined, but if this function is getting called twice
+ * with one second interval, the difference between the values returned should be equal to 1000.
```

Comment:
The `@brief` should be brief, ideally 1 sentence.  The rest should be in `@details`, `@note`, or `@remark`

```suggestion
 * @remark The moment of time where clock starts is undefined, but if this function is getting called twice
 * with one second interval, the difference between the values returned should be equal to 1000.
```